src/Microsoft.DocAsCode.Metadata.ManagedReference.Common/Filters/ConfigFilterRule.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Metadata.ManagedReference
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    using YamlDotNet.Serialization;
+
+    using Microsoft.DocAsCode.Common;
+
+    [Serializable]
+    public class ConfigFilterRule
+    {
+        [YamlMember(Alias = "apiRules")]
+        public List<ConfigFilterRuleItemUnion> ApiRules { get; set; } = new List<ConfigFilterRuleItemUnion>();
+
+        [YamlMember(Alias = "attributeRules")]
+        public List<ConfigFilterRuleItemUnion> AttributeRules { get; set; } = new List<ConfigFilterRuleItemUnion>();
+
+        public bool CanVisitApi (SymbolFilterData symbol)
+        {
+            return this.CanVisitCore(this.ApiRules, symbol);
+        }
+
+        public bool CanVisitAttribute (SymbolFilterData symbol)
+        {
+            return this.CanVisitCore(this.AttributeRules, symbol);
+        }
+
+        private bool CanVisitCore (IEnumerable<ConfigFilterRuleItemUnion> ruleItems, SymbolFilterData symbol)
```

Comment:
Remove the extra space between method name `CanVisitCore` and `(`. (Other methods have similar style issue)